frontend/app/services/personal-information-service.server.ts

```diff
@@ -10,100 +10,197 @@ import { getLogger } from '~/utils/logging.server';
+      })
+      .optional(),
+    BenefitApplicationIdentification: z
+      .object({
+        IdentificationID: z.string(),
         IdentificationCategoryText: z.string().optional(),
       })
       .array()
       .optional(),
-    PersonName: z
+    BenefitApplicationChannelCode: z
       .object({
-        PersonSurName: z.string().array().optional(),
-        PersonGivenName: z.string().array().optional(),
+        ReferenceDataID: z.string().optional(),
+        ReferenceDataName: z.string().optional(),
       })
-      .array()
       .optional(),
-    PersonContactInformation: z
+    BenefitApplicationYear: z
       .object({
-        EmailAddress: z.object({
-          EmailAddressID: z.string(),
+        BenefitApplicationYearIdentification: z.object({
+          IdentificationID: z.string(),
+          IdentificationCategoryText: z.string().optional(),
         }),
-        TelephoneNumber: z.object({
-          FullTelephoneNumber: z.string(),
-        }),
-        Address: z
-          .object({
-            AddressCategoryCode: z.object({
-              ReferenceDataName: z.string(),
-            }),
-            AddressStreet: z.object({
-              StreetName: z.string(),
-            }),
-            AddressSecondaryUnitText: z.string(),
-            AddressCityName: z.string(),
-            AddressProvince: z.object({
-              ProvinceName: z.string(),
-            }),
-            AddressCountry: z.object({
-              CountryName: z.string(),
-              CountryCode: z.string().optional(), //TODO ask if this field is a typo
-            }),
-            AddressPostalCode: z.string(),
-          })
-          .array(),
       })
       .optional(),
-    PersonLanguage: z
+    InsurancePlan: z
       .object({
-        LanguageCode: z.object({
-          ReferenceDataName: z.string(),
+        InsurancePlanIdentification: z.object({
+          IdentificationID: z.string(),
+          IdentificationCategoryText: z.string().optional(),
         }),
-        PreferredIndicator: z.boolean(),
       })
       .optional(),
-    PersonSINIdentification: z.object({
-      IdentificationID: z.string(),
-    }),
+    PrivateDentalInsuranceIndicator: z.boolean().optional(),
+    FederalDentalCoverageIndicator: z.boolean().optional(),
+    ProvicialDentalCoverageIndicator: z.boolean().optional(),
   }),
 });
 
 type PersonalInformationApiResponse = z.infer<typeof personalInformationApiResponseSchema>;
 
 const personalInfoDtoSchema = z.object({
+  applicantCategoryCode: z.string().optional(),
   applictantId: z.string().optional(),
   clientId: z.string().optional(),
+  clientNumber: z.string().optional(),
+  birthDate: z.coerce.date().optional(),
   lastName: z.string().optional(),
   firstName: z.string().optional(),
   homeAddress: z
     .object({
-      streetName: z.string(),
-      cityName: z.string(),
+      streetName: z.string().optional(),
+      secondAddressLine: z.string().optional(),
```

Comment:
We introduced a new `secondAddressLine` field for `homeAddress` and `mailingAddress` but it doesn't appear to be mapped from any value in the response. Do we need it?

Reply:
Updated the mapping (want to keep it in case we need it): 3489b449c8a8d57369f2afe860d035d0deaf0e1e